onnxruntime/test/unittest_util/checkers.cc

```diff
@@ -244,17 +254,28 @@ template <>
 struct TensorCheck<UInt4x2> {
   void operator()(const Tensor& expected, const Tensor& actual, const ValidateOutputParams& params,
                   const std::string& /*provider_type*/) const {
-    ORT_UNUSED_PARAMETER(params);
+    const bool has_abs_err = params.absolute_error.has_value();
```

Comment:
> params.absolute_error.has_value(); [](http://example.com/codeflow?start=28&length=35)

std::optional is convertible to bool, you can use it in if() clause, not need for this